components/tikv_util/src/lib.rs

```diff
@@ -62,14 +63,27 @@ pub fn panic_mark_is_on() -> bool {
 }
 
 pub const PANIC_MARK_FILE: &str = "panic_mark_file";
+pub const PANIC_MARK_FILE_CONTENT: &[u8] = b"\
+    The panic mark file is generated once TiKV encounter unrecoverable data corruption. \
+    The TiKV node will not restart. Do NOT delete the panic mark file to force TiKV \
+    restart. It would not fix the data corruption issue and could make the corruption \
+    more widespread. Please replace the TiKV node instead.\n";
```

Comment:
@Connor1996 I would like to link to a public document on how to replace the TiKV node. Any suggestion?

Reply:
If there is a panic mark, it probably due to unknown bugs. I suggest to contact us to investigate instead of replacing the node by themselves.

Reply:
Some community user may maintain the cluster themselves and sometimes would like to recover the node as soon as possible.